wsit/bundles/webservices-api-osgi/pom.xml

```diff
@@ -126,14 +126,14 @@
                             <manifestLocation>${manifest.folder}/META-INF</manifestLocation>
                             <instructions>
                                 <Export-Package>
-                                    javax.jws.*;version=2.0,
-                                    javax.xml.ws.*;version=${jaxws-api.osgiVersion},
-                                    javax.xml.soap.*;version=${saaj-api.osgiVersion}
+                                    jakarta.jws.*;version=2.0,
```

Comment:
version must be 3 here. But... I'll remove this part at some point anyway, so may be not a big deal...